app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :comments, class_name: 'Comment', foreign_key: 'author_id'
+  has_many :likes, class_name: 'Like', foreign_key: 'author_id'
+  has_many :posts, class_name: 'Post', foreign_key: 'author_id'
```

Comment:
- [ ] Since the user is the referred model with a different name which is `author`, so its associations will be forwarded to let the `User` model know that it is the `Author` in this model's queries.
So,
`has_many :comments, foreign_key: 'author_id'` Without the class_name because it won't add anything.
> So as the other two assocations